tests/integration-tests/chaos-experiments/helpers/requestCreator.js

```diff
@@ -80,10 +80,10 @@ function nockK8sChaosExperimentSupportedKinds() {
             // Additional CRDs can be listed here
         ]
     };
-
+    const status = success ? 200 : 403;
     nock('https://kubernetes').persist()
         .get('/apis/apiextensions.k8s.io/v1/customresourcedefinitions')
-        .reply(200, response);
+        .reply(status, response);
```

Comment:
response of the kins is only when success flag is true.
there should be another response for failure

Reply:
done